solution/backend/regulations/urls.py

```diff
@@ -42,5 +43,8 @@
     path('supplemental_content/<id>/', SupplementalContentView.as_view(), name='supplemental_content'),
     path('cache/', CacheView.as_view(), name='cache'),
     path('resources/', ResourcesView.as_view(), name='resources'),
-    path('statutes/', StatuteView.as_view(), name='statues')
+    path('v3/statutes', StatuteLinkConverterViewSet.as_view({
+        "get": "list",
```

Comment:
move this to the end of the list for the endpoints and do a simpilar structure to how we do it for regcore. Even though we only have 1 api for regulations currently I think it might make sense to have it separated out a little bit from views.

`path("v3/", include([
        path("statutes/", StatuteLinkConverterViewSet.as_view({
        "get": "list",),
        path("search/", include('regcore.search.urls')),`